docs/spring-boot-runtime/master.adoc

```diff
@@ -10,8 +10,8 @@ This guide covers concepts as well as practical details needed by developers to
 
 include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
+{SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, like the Embedded Tomcat Container and Java container image, when used with {SpringBoot} are provided as part of a Red Hat subscription.
```

Comment:
The word order makes the second sentence difficult to read, IMO. I would rephrase as:

Some components—for example the Embedded Tomcat Container and the Java container image—are provided as part of a Red Hat subscription when used with {SpringBoot}.